mepo.d/repository/git.py

```diff
@@ -253,14 +251,32 @@ def push(self,tags):
             cmd = self.__git + ' push -u {}'.format(self.__remote)
         return shellcmd.run(cmd.split(), output=True).strip()
 
-    def get_remote_latest_commit_id(self, branch):
-        cmd = self.__git + ' ls-remote {} refs/heads/{}'.format(self.__remote, branch)
-        output = shellcmd.run(cmd.split(), output=True).strip()
-        if not output:
-            msg = 'Branch {} does not exist on {}'.format(branch, self.__remote)
-            msg += " Have you run 'mepo push'?"
-            raise RuntimeError(msg)
-        return output.split()[0]
+    def get_remote_latest_commit_id(self, branch, commit_type):
```

Comment:
For your amusement:   I misread this as `git_remove_latest_commid_id` at first and was alarmed.